doc/user/content/sql/create-source/postgres.md

```diff
@@ -189,12 +189,11 @@ i.e. in Materialize, the data will not be returned when serving `SELECT`s from
 the inherited table.
 
 You can mimic PostgreSQL's `SELECT` behavior with inherited tables by creating a
-materialized view that `UNION`s data from the inherited and inheriting tables,
+materialized view that `UNION ALL`s data from the inherited and inheriting tables,
 though there are many caveats:
 
 -   Materialized views are maintained in arrangements, which moves the tables'
-    data from being stored in relatively low-cost storage into being stored in
-    memory.
+    data from being stored in relatively low-cost storage into memory.
```

Comment:
Only thought is that if we are polishing this, we should maybe instead delete it because materialized views are no longer stored in arrangements?

Reply:
Perhaps we would want to introduce a different caveat, that there will be ongoing compute work to maintain the materialized view. Though perhaps a non-materialized view is not out of scope either? Not really sure!